pkgs/applications/version-management/git-and-tools/git-big-picture/default.nix

```diff
@@ -0,0 +1,23 @@
+{ fetchgit, python2Packages, stdenv, git, graphviz }:
+
+python2Packages.buildPythonApplication rec {
+    pname = "git-big-picture";
+    version = "0.9.0";
+
+    name = "${pname}-${version}";
+
+    src = fetchgit {
+      url = "https://github.com/esc/${pname}.git";
+      rev = "fbe3b9504e255da859067fd58e90d849d63e5381";
+      sha256 = "1h283gzs4nx8lrarmr454zza52cilmnbdrqn1n33v3cn1rayl3c9";
+    };
+
+  propagatedBuildInputs = [ git graphviz ];
```

Comment:
You want these to be available during runtime, not propagate them to packages depending on this package.
You'll want to wrap binaries created by this package and put `git` and `graphviz` on `PATH`. Check `makeBinPath`.

(I think I know why you add them as `propagatedBuildInputs` because that's what we do with Python packages for run-time dependencies. Its not the same.)